common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -232,4 +223,17 @@ private synchronized void loadServerIcon(ResourceLocation destination) {
             }
         }
     }
+
+    private enum ServerType {
+        LOBBY("lobby"),
+        GAME("game"),
```

Comment:
I'm pretty sure the official address is `play`. I think we should use that.

Reply:
```suggestion
        GAME("play"),
```